src/CoreVideo/CVDisplayLink.cs

```diff
@@ -192,20 +251,43 @@ public CVReturn GetCurrentTime (out CVTimeStamp outTime)
 		static CVReturn OutputCallback (IntPtr displayLink, ref CVTimeStamp inNow, ref CVTimeStamp inOutputTime, CVOptionFlags flagsIn, ref CVOptionFlags flagsOut, IntPtr displayLinkContext)
 		{
 			GCHandle callbackHandle = GCHandle.FromIntPtr (displayLinkContext);
-			DisplayLinkOutputCallback func = (DisplayLinkOutputCallback) callbackHandle.Target;
+			DisplayLinkOutputCallback func = (DisplayLinkOutputCallback) callbackHandle.Target!;
 			CVDisplayLink delegateDisplayLink = new CVDisplayLink(displayLink, false);
 			return func (delegateDisplayLink, ref inNow, ref inOutputTime, flagsIn, ref flagsOut);
 		}
 	  
 		[DllImport (Constants.CoreVideoLibrary)]
 		extern static CVReturn CVDisplayLinkSetOutputCallback (IntPtr displayLink, CVDisplayLinkOutputCallback function, IntPtr userInfo);
+
 		public CVReturn SetOutputCallback (DisplayLinkOutputCallback callback)
 		{
 			callbackHandle = GCHandle.Alloc (callback);
 			CVReturn ret = CVDisplayLinkSetOutputCallback (this.Handle, static_OutputCallback, GCHandle.ToIntPtr (callbackHandle));
 				
 			return ret;
 		}
+
+		[Mac (12,0), NoiOS, NoTV]
+		[DllImport (Constants.CoreVideoLibrary)]
+		static extern nuint CVDisplayLinkGetTypeID ();
+
+		[Mac (12,0), NoiOS, NoTV]
+		public static nuint GetTypeId ()
+			=> CVDisplayLinkGetTypeID ();
+
+		[Mac (12,0), NoiOS, NoTV]
+		[DllImport (Constants.CoreVideoLibrary)]
+		static extern int CVDisplayLinkTranslateTime (IntPtr displayLink, CVTimeStamp inTime, ref CVTimeStamp outTime);
+
+		[Mac (12,0), NoiOS, NoTV]
+		public bool TryTranslateTime (CVTimeStamp inTime, [NotNullWhen (true)] out CVTimeStamp outTime)
+		{
+			outTime = new CVTimeStamp() { Version = 0 };
```

Comment:
This is slightly simpler:

```suggestion
			outTime = default (CVTimeStamp);
```